src/rules/function-url-quotes/index.js

```diff
@@ -54,19 +61,25 @@ export default function (expectation) {
     }
 
     function checkArgs(args, node, index, functionName) {
+      let shouldHasQuotes = expectation === "always"
+
       const leftTrimmedArgs = args.trimLeft()
       if (!isStandardSyntaxUrl(leftTrimmedArgs)) { return }
       const complaintIndex = index + args.length - leftTrimmedArgs.length
       const hasQuotes = leftTrimmedArgs[0] === "'" || leftTrimmedArgs[0] === "\""
-      switch (expectation) {
-        case "always":
-          if (hasQuotes) { return }
-          complain(messages.expected(functionName), node, complaintIndex)
-          return
-        case "never":
-          if (!hasQuotes) { return }
-          complain(messages.rejected(functionName), node, complaintIndex)
-          return
+
+      const trimmedArg = args.trim()
+      const isEmptyArgument = [ "", "'", "\"\"" ].includes(trimmedArg)
```

Comment:
I do not understand this line. It does not seem like it is in fact checking if the argument is empty. What's the intention?